src/main/resources/com/sonyericsson/hudson/plugins/gerrit/trigger/diagnostics/EventListenersReport/index.groovy

```diff
@@ -0,0 +1,166 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2016 CloudBees Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.sonyericsson.hudson.plugins.gerrit.trigger.diagnostics.EventListenersReport
+
+import com.sonyericsson.hudson.plugins.gerrit.trigger.GerritManagement
+import com.sonyericsson.hudson.plugins.gerrit.trigger.PluginImpl
+import com.sonyericsson.hudson.plugins.gerrit.trigger.diagnostics.Diagnostics
+import com.sonyericsson.hudson.plugins.gerrit.trigger.hudsontrigger.EventListener
+import com.sonyericsson.hudson.plugins.gerrit.trigger.diagnostics.EventListenersReport
+import com.sonyericsson.hudson.plugins.gerrit.trigger.hudsontrigger.GerritTrigger
+import hudson.model.Job
+
+EventListenersReport report = my;
+
+def l = namespace(lib.LayoutTagLib)
+
+l.layout(title: _("${report.getDisplayName()} - Gerrit Trigger Diagnostics"), norefresh: false, permission: Diagnostics.requiredPermission) {
+    l.header {
+        style {
+            raw("""
```

Comment:
Mmm, I'm going to go for the inline bash script in the cje-war pipeline.   This is not so bad to read,  for me.